kitsune/wiki/views.py

```diff
@@ -125,6 +128,30 @@ def document(request, document_slug, template=None, document=None):
             # and OK to fall back to parent (parent is approved).
             fallback_reason = 'no_translation'
 
+    # If any document is not translated into requested locale, we will try to find out the
```

Comment:
Entire section rewrite:
# Find and show the defined fallback locale rather than the English version of the document
# The fallback locale is defined based on the ACCEPT_LANGUAGE header, site-wide locale mapping and custom fallback locale
# The custom fallback locale is defined in the FALLBACK_LOCALES array in kitsune/wiki/config.py
# See bug 800880 for more details